torch/distributed/nn/api/remote_module.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/python3
+import uuid
+from typing import Any, Callable, Dict, Tuple
+
+import torch
+import torch.distributed.rpc as rpc
+from torch.distributed.nn.jit import instantiator
+
+
+def _gen_global_unique_name():
+    return f"{uuid.uuid4().hex}"
+
+
+def _instantiate_template(global_unique_name, module_interface_cls, is_scriptable):
+    # Generate the template instance name.
+    generated_module_name = f"_remote_module_{global_unique_name}"
+
+    # Instantiate _RemoteModule class template on the local side.
+    generated_module = instantiator.instantiate_remote_module_template(
+        generated_module_name, module_interface_cls, is_scriptable
+    )
+
+    remote_module_cls = generated_module._RemoteModule
+
+    return remote_module_cls
+
+
+def _script_module_creator_wrapper(module_creator, module_interface_cls, args, kwargs):
+    module = module_creator(*args, **kwargs)
+    script_module = torch.jit.script(module)
+    return rpc.RRef(script_module, module_interface_cls)
+
+
+def RemoteModule(
+    to: str,
+    module_creator: Callable,
+    args: Tuple = None,
+    kwargs: Dict[str, Any] = None,
+    global_unique_name: str = None,
+    module_interface_cls=None,
```

Comment:
space around "="

Reply:
Let me annotate as Any type to align them.